src/logging/Logger.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
do we need this?

Reply:
There is an eslint rule that warns about the use of the `any` type (so that we actually have to type things rather than just making everything `any` which matches anything). Here we are using `any` legitimately though to say that we will allow anything to be passed - so this comment just tells eslint to ignore that rule for the following line.

Reply:
ohhh I see. thank you!